homeassistant/components/homewizard/switch.py

```diff
@@ -111,7 +113,7 @@ def __init__(
         super().__init__(coordinator, entry, "switch_lock")
 
         # Config attributes
-        self._attr_name = f"{entry.title} Switch Lock"
+        self._attr_name = "Switch lock"
```

Comment:
As this is now a fixed value, it can be moved up as a class attribute (just like `_attr_device_class` above).